kpops/cli/pipeline_config.py

```diff
@@ -107,8 +107,8 @@ def customise_sources(
         ):
             return (
                 init_settings,
-                yaml_config_settings_source,
                 env_settings,
```

Comment:
shouldn't `env_settings` be at the very top

Reply:
Moved it.